src/trusted/service_runtime/sel_ldr.c

```diff
@@ -666,17 +680,17 @@ int NaClAddThread(struct NaClApp        *nap,
 
 void NaClRemoveThreadMu(struct NaClApp  *nap,
                         int             thread_num) {
-  if (NULL == DynArrayGet(&nap->threads, thread_num)) {
-    NaClLog(LOG_FATAL,
+  if (!DynArrayGet(&nap->threads, thread_num)) {
+     NaClLog(LOG_FATAL,
             "NaClRemoveThreadMu:: thread to be removed is not in the table\n");
   }
   if (nap->num_threads == 0) {
-    NaClLog(LOG_FATAL,
+     NaClLog(LOG_FATAL,
             "NaClRemoveThreadMu:: num_threads cannot be 0!!!\n");
   }
   --nap->num_threads;
   if (!DynArraySet(&nap->threads, thread_num, (struct NaClAppThread *) NULL)) {
-    NaClLog(LOG_FATAL,
+     NaClLog(LOG_FATAL,
             "NaClRemoveThreadMu:: DynArraySet at position %d failed\n",
             thread_num);
```

Comment:
@alyptik noticed some whitespacing inconsistencies here.

Reply:
This change was intended; the original was indented with one space incorrectly.